dakara_server/library/management/commands/feed_components/default_work_parser.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+
+def parse_work(filepath):
+    """Default parse module for work data file in JSON.
+
+    Args:
+        filepath : path of the file to parse (must be JSON).
+    """
+
```

Comment:
I would at least check if the file exists, so as to have a custom error message instead of a generic one in this not-that-improbable case.